### ISSUE TITLE
Image in avatarify camera doesn't flow smoothly

### ISSUE THREAD

**Issue by darellwest** [alievk/avatarify-python#353]
I tried installing avatarify on my pc everything works while following the tutorial except for the fact that the image in the camera
doesn't move smoothly. I don't know much about python and I get these error notifications below.

C:\Users\ewere\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:3385: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  warnings.warn("Default grid_sample and affine_grid behavior has changed "
C:\Users\ewere\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1639: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")

Please i need your help.
 I am on windows
16gig ram
 GTX 1660 Ti

**Comment by JohanAR** [comment]
Did you have a chance to try alievk's suggestion in the other thread? https://github.com/alievk/avatarify/issues/349#issuecomment-764652671

If Open Avatarify (this Python program) gives you a lot of trouble, maybe you'd be interested in giving Avatarify Desktop (Windows app) a try? https://github.com/alievk/avatarify/wiki/Windows-App-(Beta)

**Comment by JohanAR** [comment]
Haven't heard anything in a while, but the Avatarify Windows app has now been released as [Avatarify Desktop](https://github.com/alievk/avatarify-desktop). Please give it a try if Avatarify Python is giving you problems.

**Comment by Elujekwuteben** [comment]
That is easy to solve.
I can put you through on steps to upgrade your fps speed and your camera won't lag again.
Just DM on Whatsapp
+2349037013612
